tests/test_opensubtitles.py

```diff
@@ -260,3 +260,15 @@ def test_download_subtitle(movies):
     assert subtitles[0].content is not None
     assert subtitles[0].is_valid() is True
     assert subtitles[0].encoding == 'cp1252'
+
+
+@pytest.mark.integration
+@vcr.use_cassette
+def test_tag_match(episodes):
+    video = episodes['the fall']
+    languages = {Language('por', 'BR')}
+    unwanted_subtitles = {'1954369181'}
+    with OpenSubtitlesProvider() as provider:
+        subtitles = provider.list_subtitles(video, languages)
+    assert len(subtitles) > 0
+    assert {subtitle.id for subtitle in subtitles} != unwanted_subtitles
```

Comment:
@Diaoul not sure if is the best unittest. Its the first time i did one. so please bear with me.

My goal is to test that: this wrong subtitle `1954369181` shouldn't be in the list of results because even if is a tagmatch, the imdb is wrong.